src/components/ExtraAnnotations/index.tsx

```diff
@@ -88,7 +88,8 @@ const ExtraAnnotations: React.FC<ExtraAnnotationsProps> = ({ namespace }) => {
     []
   );
 
-  if (loading && !isStale) return <Loading size="small" />;
+  const loading = !data;
```

Comment:
does this mean if the `data` is empty (because there are no annotations`, the loading spinner will show forever?

Reply:
@SandyRogers no, when the data is empty the spinner does not keeo showing. 
An empty array evaluates to true.

Reply:
I also tried this on pages that have empty data and it works fine